packages/firestore/src/local/remote_document_change_buffer.ts

```diff
@@ -78,17 +78,17 @@ export class RemoteDocumentChangeBuffer {
    * the provided transaction.
    */
   apply(transaction: PersistenceTransaction): PersistencePromise<void> {
-    const changes = this.assertChanges();
+    const docs: MaybeDocument[] = [];
```

Comment:
Would it make any sense to have RemoteDocumentCache.addEntries() accept a MaybeDocumentMap so we don't need to do this MaybeDocumentMap => array of documents conversion?  I don't care strongly.

Reply:
I'm leaning to leave it as is for now. `getChanges` returns an array of MaybeDocuments[] and matches this method.